indico/modules/rb/client/js/setup.jsx

```diff
@@ -33,14 +34,7 @@ export default function setup(overrides = {}, postReducers = []) {
 
   document.addEventListener('DOMContentLoaded', () => {
     const appContainer = document.getElementById('rb-app-container');
-    const store = createRBStore(overrides, postReducers);
-
-    // Use this function whenever you would like to override
-    // a component e.g. from within a plugin without
-    // using the `setup` function
-    window.registerOverride = externalOverrides => {
```

Comment:
I removed this for now since we:

- don't actually use it
- will probably need react state to store the overrides once we want to add something like this back

Reply:
Maybe we were using this in the shower site? 😆

Reply:
nope, that's using `setup()`